api/controllers/auth/reset-password-verify.js

```diff
@@ -49,11 +50,14 @@ module.exports = async function (req, res) {
 
       // redirect to our base url
       res.redirect("/");
+      authLogger(req, "Password reset token successful");
    } catch (err) {
       if (err.code == "EUNKNOWNTOKEN") {
+         authLoogger(req, "Password reset token FAILED");
```

Comment:
should this be `authLogger`  (<-- only one `o` )?

Reply:
Thanks for catching that